.github/workflows/apply_spotless.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Apply Spotless
+
+on:
+  pull_request:
+    branches: [ '*' ]
```

Comment:
Not that it matters, but I'm pretty sure this line is redundant

Reply:
Yeah it looks like it still runs without it. Thanks!